Whiskey/Tasks/Invoke-WhiskeyProcess.ps1

```diff
@@ -58,11 +73,22 @@ function Invoke-WhiskeyProcess
     $path = $TaskParameter['Path']
     if ( -not $path )
     {
-        Stop-WhiskeyTask -TaskContext $TaskContext -Message ('Property ''Path'' is mandatory. It should be the Path to the executable you want to start the Process with.')
+        Stop-WhiskeyTask -TaskContext $TaskContext -Message ('Property ''Path'' is mandatory. It should be the Path to the executable you want to start the Process with, e.g.
+        
+            BuildTasks:
+            - Process:
+                Path: cmd.exe
+            
+        ')
     }
 
     $processPath = ''
-    if ( Test-Path -Path $path -PathType Leaf )
+    $pathRelativeToBuildRoot = Join-Path -Path $TaskContext.BuildRoot -ChildPath $path

```

Comment:
This is getting hard for me to understand. Does this work (replacing lines 85 to 104)?

```
if( -not [IO.Path]::IsPathRooted($path) )
{
    $path = Join-Path -Path $TaskContext.BuildRoot -ChildPath $path
}

if( (Test-Path -Path $path -PathType Leaf) )
{
    $path = $path | Resolve-Path | Select-Object -ExpandProperty 'ProviderPath'
}
else
{
    $path = $TaskParameter['Path']
    if( -not (Get-Command -Name $path -CommandType Application -ErrorAction Ignore) )
    {
        Stop-WhiskeyTask -TaskContext $TaskContext -Message ('Executable ''{0}'' does not exist. We checked if the executable is at that path on the file system and if it is in your PATH environment variable.' -f $path)
    }
}
```

Reply:
Much more concise, I like it.